prime_factor/prime_factor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+
+void generate_primes(int *ar, int *cnt) {
```

Comment:
Sir, as u see it is just generating primes and not counting the primes for that no. It is handled in a different function. It is working on reference so it automatically updates it in the previous function.😁